filament/backend/src/opengl/PlatformEGL.cpp

```diff
@@ -292,6 +299,9 @@ Driver* PlatformEGL::createDriver(void* sharedContext) noexcept {
 
     initializeGlExtensions();
 
+    // this is needed with older emulators/API levels on Android
```

Comment:
uh?! so some GL calls fail before we get here?

Reply:
On the emulator with ES 3.0 enabled and the KitKat system image, enumerating the GL extensions generates a GL error in the log (in the emulator driver, so not a bug on our side). ES 3.0 was just incomplete.